component/authenticator/src/main/java/org/wso2/carbon/identity/policy/password/PasswordChangeUtils.java

```diff
@@ -74,9 +74,16 @@ public static void loadProperties() {
      */
     public static int getPasswordExpirationInDays() {
         if (properties.get(PASSWORD_EXP_IN_DAYS) != null) {
-            return Integer.parseInt((String) properties.get(PASSWORD_EXP_IN_DAYS));
-        } else {
-            return DEFAULT_PASSWORD_EXP_IN_DAYS;
+            String passwordExpPropertyValue = (String) properties.get(PASSWORD_EXP_IN_DAYS);
+            try {
+                return Integer.parseInt(passwordExpPropertyValue);
+            } catch (NumberFormatException e) {
+                log.warn(String.format("Invalid value: %s for property %s. Returning default password " +
+                                       "expiration time: %d days.", passwordExpPropertyValue, PASSWORD_EXP_IN_DAYS,
```

Comment:
Shall we say that days need to configured in intergers?

Reply:
Fixed